src/preview/upload.rs

```diff
@@ -70,15 +69,15 @@ pub fn upload(
                         message::info("Uploading updated files...");
                     }
 
-                    upload_files(target, user, &site_namespace.id, to_upload)?;
+                    bulk::put(target, user, &site_namespace.id, to_upload, &None)?;
```

Comment:
should  we use a progress bar here? didn't before but can add.

Reply:
That would make sense to me, right now it just says nothing and it might look like it's hanging